docs/en/writing-running-appium/touch-actions.md

```diff
@@ -189,6 +189,14 @@ $params = array(array('direction' => 'down', 'element' => element.GetAttribute("
 $driver->executeScript("mobile: scroll", $params);
 ```
 
+**Swiping**
+
+This is an XCTest specific method that is similar to scrolling (for reference, see 
```

Comment:
Maybe instead of `XCTest` say `XCUITest driver` to tie it more strongly to how we call things on the Appium side? Otherwise LGTM